### DISCUSSION TITLE
【求助】帮我做出选择

### DISCUSSION THREAD

**Discussion by jidianzhuxing** [share121/inter-knot#4853] [崩坏：星穹铁道]
如图这是我的角色池，这期UP我是应该抽鸟还是飞霄，目前75抽大保底
![屏幕截图 2024-09-14 134636](https://github.com/user-attachments/assets/cd6ae335-5e4e-4371-9637-866edb7a0a14)
![屏幕截图 2024-09-14 134658](https://github.com/user-attachments/assets/75eb8e5e-bc70-43ba-a4c3-58d07b9a0d53)
![屏幕截图 2024-09-14 134721](https://github.com/user-attachments/assets/a098cb5b-151f-4bf5-962a-e8583908c2a3)
真的不知道该抽谁。

**Comment by Pleasurecruise** [comment]
飞霄

**Comment by InkSummer** [comment]
还是飞霄吧，你没抽黄泉流萤这些大C，辅助再多也不如大C强，而且飞霄打模拟宇宙抓猪打罐子也有奇效

**Comment by HLXS-HLXS** [comment]
飞霄吧，你需要个c，但马上3.0了，有可能像2.0一样把上一个大版本的角色都淘汰了，不急等等看
